Program.cs

```diff
@@ -48,27 +50,30 @@ class Program
 	{
 		private static EventWaitHandle waitHandle;
 
+		private const int RaygunMaxNotifications = 15;
+
+		private static int raygunNotificationCount = 0;
+
+		private static RaygunClient _raygunClient;
+
 		private static string mainServiceName = "";
 
 		[STAThread]
 		static void Main(string[] args)
 		{
-			// this sets the global culture for the app and all new threads
+			// Set the global culture for the app, current thread and all new threads
```

Comment:
@larsbrubaker  - any reason to not bring these two Thread Culture assignments together?